### PR TITLE
docs: add metrics configuration

### LINKED ISSUE #431: Document how to use Rollkit metrics
Similar to https://docs.cometbft.com/v0.37/core/metrics

### PR DESCRIPTION
**Pull request by tuxcanfly**
## Overview

Fixes #431 

## Release Notes

- **New Features**
	- Added a section on configuring Prometheus metrics in the documentation.
	- Introduced a guide for running a centralized sequencer.
	- Listed available metrics with descriptions and types in the documentation.

- **Documentation**
	- Enhanced navigation with new sidebar items for the centralized sequencer and Prometheus metrics.
	- Updated overview guide to include new entries for better user resource access.

### CODE REVIEW

**Comment by MSevey** [comment]
docs preview is failing: https://github.com/rollkit/docs/actions/runs/11072258066/job/30766061486?pr=468
might not be related to this PR. Will investigate

**Comment by MSevey** [comment]
Tracking build failure here: https://github.com/rollkit/docs/pull/469

File: guides/metrics.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Metrics
+
+Rollkit can report and serve the Prometheus metrics, which in their turn can
+be consumed by Prometheus collector(s).
+
+This functionality is disabled by default.
+
+To enable the Prometheus metrics, set `instrumentation.prometheus=true` in your
+config file. Metrics will be served under `/metrics` on 26660 port by default.
```
Comment (tzdybal) [inline guides/metrics.md:11]: Maybe a dumb question - but which file exactly?

Comment (MSevey) [reply guides/metrics.md:11]: rollkit.toml?

**Review by MSevey — CHANGES_REQUESTED** [review]
The page isn't linked to the site yet https://rollkit.github.io/docs-preview/pr-468/guides/overview

You need to add it to the menu.

File: guides/metrics.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+| p2p\_num\_txs                              | Gauge     | peer\_id         | Number of transactions submitted by each peer\_id                      |
+| p2p\_pending\_send\_bytes                  | Gauge     | peer\_id         | Amount of data pending to be sent to peer                              |
+| mempool\_size                              | Gauge     |                  | Number of uncommitted transactions                                     |
+| mempool\_tx\_size\_bytes                   | Histogram |                  | Transaction sizes in bytes                                             |
+| mempool\_failed\_txs                       | Counter   |                  | Number of failed transactions                                          |
+| mempool\_recheck\_times                    | Counter   |                  | Number of transactions rechecked in the mempool                        |
+| state\_block\_processing\_time             | Histogram |                  | Time between BeginBlock and EndBlock in ms                             |
```
Comment (tzdybal) [inline guides/metrics.md:54]: Is this list autogenerated? How can we ensure it's up to date?

File: guides/metrics.md (hunk 1 above)
Comment (tuxcanfly) [reply guides/metrics.md:11]: Actually this is the cometbft app's `config.toml`. Will clarify the location 👍🏽

File: guides/metrics.md (hunk 2 above)
Comment (tuxcanfly) [reply guides/metrics.md:54]: Will rewrite this to link to cometbft for default metrics and only document metrics exposed by rollkit specifically 👍🏽

**Review by MSevey — APPROVED** [review]

**Review by Manav-Aggarwal — APPROVED** [review]

**Pull request merged**